polyinterface/polylogger.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import logging
+from logging import handlers as log_handlers
+import warnings
+
+class PolyLogger:
+
+    NAME = __name__.split(".")[0]

```

Comment:
I'm not sure about purpose of this. I believe __name__ here will resolve to "polylogger", that is, module name, rather than the name of executable that invoked. Was that the intent? If so, there's no need for split.

Reply:
No, __name__ is polyinterface.polylogger so I just wanted polyinterface which was what you get when it's inside __init__ like I had before.  I could leave it hardcoded like you had it, but this works as well.

Reply:
It's fine. Just slightly confusing :)